cimr/processor/harmon.py

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
What is this module for? I didn't it being used anywhere. Is it going to be used in the future?

Reply:
Yes... it had some additional functions I was testing but I didn't want to include it in this PR so I deleted it. I left the file as a reminder to myself but I can delete it for now.